tools/batch_translate.py

```
@@ -0,0 +1,221 @@
+            # Find all markdown files in directory
+            md_files = list(path.rglob('*.md'))
+            
+            if not include_bundles:
+                # Filter out index.md files (page bundles)
+                md_files = [f for f in md_files if f.name != 'index.md']
+            
+            files.extend(md_files)
+    
+    return files
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)